<?xml version="1.0" ?>
<launch>

  <arg name="plc_ip_addr" default="192.168.0.123"/>
  <arg name="respawn" default="true" />

  <param name = "radius" type = "double" value = "0.075" />
  <param name = "wheel_dist" type = "double" value = "0.435" />
  <param name = "odom_pub_rate" type = "double" value = "10" />
  <param name = "plc_ip_addr" type = "string" value = "$(arg plc_ip_addr)"/>
  <param name = "motor1_addr" type = "string" value = "MD6.0" />
  <param name = "motor2_addr" type = "string" value = "MD10" />
  <param name = "encoder1_addr" type = "string" value = "MD20" />
  <param name = "encoder2_addr" type = "string" value = "MD28" />

  <!-- run teleop_plc -->  
  <node pkg="mw_seimens_s7_plc_interface" type="siemens_plc_controller.py" name="siemens_plc_controller" output="screen" respawn="$(arg respawn)"/>


</launch>
